### PR TITLE
BETA: Register polar.is-a.dev

### PR DESCRIPTION
**Pull request by polarxpression**
Added `polar.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/polar.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "polarfreez",
+        "email": "rafaelbraun5@hotmail.com"
+    },
+    "record": {
+        "CNAME": "https://polarfreez.github.io"
```
Comment (CuteDog5695) [inline domains/polar.json:7]: CNAME cannot have https:// (also site gives 404)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/polar.json (hunk 1 above)
Comment (polarfreez) [reply domains/polar.json]: Fixed. Can you please check again?

**Comment by polarfreez** [comment]
Fixed. Can you please check again?

**Review by iostpa — APPROVED** [review]
think its fine to me now

**Pull request merged**